runelite-client/src/main/java/net/runelite/client/plugins/skillcalculator/SkillCalculatorPlugin.java

```diff
@@ -82,4 +85,10 @@ protected void shutDown() throws Exception
 	{
 		clientToolbar.removeNavigation(uiNavigationButton);
 	}
+
+	@Schedule(period = 2, unit = ChronoUnit.SECONDS)
+	public void updateSkillCalculator()
```

Comment:
There is an xp changed event which might be better here.

Reply:
Thanks - i'll take a look